packages/react-components/src/components/VideoTile.tsx

```diff
@@ -401,7 +438,22 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
             )}
           </Stack>
         )}
-
+        
+        { /* @conditional-compile-remove(reaction) */
+        reaction?.shouldRender && (currentUnitTimestamp - reaction.receivedTimeStamp) < 3000 && (
```

Comment:
We should just check to see that the resource has loaded for the sprite map. if this is the only reason this state is on the reaction type we should remove it and make sure that we are getting the sprite map or however the reactions will be loaded.